src/Umbraco.Web.BackOffice/Trees/TreeNodesRenderingNotification.cs

```diff
@@ -26,11 +26,17 @@ public class TreeNodesRenderingNotification : INotification
         /// </summary>
         public string TreeAlias { get; }
 
-        public TreeNodesRenderingNotification(TreeNodeCollection nodes, FormCollection queryString, string treeAlias)
+        /// <summary>
+        /// Gets id of the node rendered
+        /// </summary>
+        public string Id { get; }
+
+        public TreeNodesRenderingNotification(TreeNodeCollection nodes, FormCollection queryString, string treeAlias, string id)
```

Comment:
@nzdev If you can leave the old constructor in place and obsolete it then this additional constructor is fine, but as it is now, this would be a breaking change.

Reply:
All done :)